doc/users/next_whats_new/delete_Bracket_and_update_Curve.rst

```diff
@@ -0,0 +1,16 @@
+Deleted the _Bracket and updated _Curve
+---------------------------------------
+
+The updated _Curve now has the ability of doing the same things the old
```

Comment:
Don't include private functions in the public API discussion!

Reply:
:facepalm: Sorry, forgot that!